ui/app/AppLayouts/Chat/ChatColumn/ChatComponents/ChatCommandModal.qml

```diff
@@ -60,6 +60,17 @@ ModalPopup {
                 anchors.topMargin: 19
                 icon.rotation: root.isRequested ? -90 : 90
             }
+
+            StyledText {
+                id: addressRequiredInfo
+                anchors.right: selectRecipient.right
+                anchors.bottom: selectRecipient.top
+                anchors.bottomMargin: -Style.current.padding
+                text: qsTr("Address request required")
+                color: Style.current.danger
+                visible: !root.isRequested
+            }
```

Comment:
When this modal originates from a "send transaction" command, we can always show this info because addresses need to be requested every time.

Reply:
In the case of the ENS flow, it does not.

Reply:
Should this be set to addressRequestedValidator.isWarn?